src/dotnet/commands/dotnet-add/dotnet-add-reference/Program.cs

```diff
@@ -91,7 +91,7 @@ public override int Execute()
             }
 
             var relativePathReferences = _appliedCommand.Arguments.Select((r) =>
-                                                                              PathUtility.GetRelativePath(msbuildProj.ProjectDirectory, Path.GetFullPath(r)))
+                                                                              Path.GetRelativePath(msbuildProj.ProjectDirectory, Path.GetFullPath(r)))
```

Comment:
Do we still need to keep the PathUtility.GetRelativePath around after your change? Just curious, we don't need to block this PR on deleting that code, if possible.

Reply:
TestFramework code is still using it and targeting netstandard (which doesn't have BCL equivalent yet.) We can clean it up separately.